pkgs/games/path-of-building/default.nix

```diff
@@ -59,6 +97,27 @@ stdenv.mkDerivation {
     )
   '';
 
+  desktopItems = [
+    (makeDesktopItem {
+      name = "Path of Building";
```

Comment:
The name should probably be `path-of-building` or `pobfrontend` or something without spaces.

Reply:
Ok, done.